test/api/capi/capi_table_functions.cpp

```diff
@@ -65,6 +65,7 @@ static void capi_register_table_function(duckdb_connection connection, const cha
 	// add a string parameter
 	duckdb_logical_type type = duckdb_create_logical_type(DUCKDB_TYPE_BIGINT);
 	duckdb_table_function_add_parameter(function, type);
+	duckdb_table_function_add_named_parameter(function, "my_parameter", type);
```

Comment:
Could you add a test where the named parameter is used in a SQL query?

Reply:
Will do!

Reply:
Updated with an additional test.